RELEASING.md

```diff
@@ -33,7 +33,9 @@ release is announced.
     ```
 1. Update the `RELEASE_STREAM` variable at the top of calico_node/Makefile to the new major/minor version.
 1. Test the changes locally then open a pull request, make sure it passes CI and get it reviewed.
-1. Create a git tag, docker images, and release file by running `make -C calico_node release`. Follow the instructions to push the images but DO NOT PUSH THE TAG YET.
+1. Run `make -C calico-node release` - **do not push the git tag yet** - and follow the instructions to:
+   1. Create release docker images (do not push the `latest` tag yet).
+   2. Create the release file
```

Comment:
Can you mark this step as `1.` so github handles the number ordering itself?

Reply:
fixed in latest push